Formula/openjdk.rb

```diff
@@ -1,8 +1,9 @@
 class Openjdk < Formula
   desc "Development kit for the Java programming language"
   homepage "https://openjdk.java.net/"
-  url "https://github.com/openjdk/jdk18u/archive/jdk-18.0.2.1-ga.tar.gz"
-  sha256 "06fad73665af281e36e1cc5fb0c8ed5e88e1e821989f1421539cb012065d7722"
+  url "https://github.com/openjdk/jdk19u/archive/jdk-19+36.tar.gz"
+  version "19"
```

Comment:
May wait a few days to see if upstream creates a `jdk-19-ga` tag. Looks like OpenJDK/Oracle/etc have already built v19 binaries off the build version.

https://jdk.java.net/19/release-notes
> The full version string for this release is build 19+36 (where "+" means "build"). The version number is 19.

Reply:
Also https://mail.openjdk.org/pipermail/jdk-dev/2022-September/006933.html (emphasis mine):
> JDK 19, the reference implementation of Java 19, is now Generally
Available.  We shipped build 36 as the first Release Candidate of
JDK 19 on 18 August, and no P1 bugs have been reported since then.
**Build 36 is therefore now the GA build, ready for production use.**